wwwapp/settings.py

```diff
@@ -233,11 +233,18 @@
     'height': 500,
     'branding': False,
     'image_advtab': True,
-    'paste_data_images': True,
+    'paste_data_images': False,
     'relative_urls': False,
     'remove_script_host': True,
     'link_list': '/articleNameList/',
 }
+TINYMCE_DEFAULT_CONFIG_WITH_IMAGES = {  # Additional settings for editors where image upload is allowed
+    'plugins': 'preview paste searchreplace autolink code visualblocks visualchars image link media codesample table charmap hr nonbreaking anchor toc advlist lists wordcount imagetools textpattern quickbars emoticons',
```

Comment:
can't we define `plugins` and `toolbar` in terms of `TINYMCE_DEFAULT_CONFIG['plugins']` & toolbar fields?

Reply:
`plugins` - probably yes
`toolbar` - not really since the order defines where they are displayed
So I chose the ugly copy-paste to keep them both the same

Reply:
OK!